<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);
border-image: none;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startWindow_CreateFileBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>95</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>95</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>95</width>
      <height>45</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	padding-bottom: 3px;
	color: white;
	border: none;
	background-color:rgb(56, 76, 255)
}

QPushButton::hover {
	background-color: rgb(34, 67, 255)
}</string>
    </property>
    <property name="text">
     <string>â®œ  BACK</string>
    </property>
   </widget>
   <widget class="QFrame" name="createFileWindow_HeadingFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>661</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="createFileWindow_HeadingLabel">
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Create A File.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="createFileWindow_TextFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>199</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="createFileWindow_TextLabel">
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>To create a file, enter in the name of the file and select a location you would like to store the file.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="createFileWindow_FileNameFrame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>339</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="createFileWindow_FileNameLabel">
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Enter File Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="createFileWindow_FileNameLineEdit">
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	padding-left: 10px;
	padding-right: 10px;
	background-color: white;
	color: black;
	border: none;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="createFileWindow_ExtensionsComboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
	padding-left: 10px;
	padding-right: 10px;
	background-color: white;
	border: none;
}</string>
         </property>
         <item>
          <property name="text">
           <string>- - - -</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>class</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>css</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>html</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>java</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>js</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>json</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>jsx</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>md</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>module.scss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>php</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>py</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>scss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>swift</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ts</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>txt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vb</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="createFileWindow_SelectedFolderFrame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>439</y>
      <width>571</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="createFileWindow_SelectedFolderLabel">
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Choose location:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="createFileWindow_SelectedFolderLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgb(188, 188, 188);
	color: rgb(127, 127, 127);
	border: none;
	padding-left: 10px;
	padding-right: 10px;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createFileWindow_SelectedFolderBtn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: white;
	background-color: black;
	padding-bottom: 5px;
}</string>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="createFileWindow_CreateFileBtnFrame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="createFileWindow_CreateFileBtn">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: white;
	border: none;
	background-color:rgb(94, 137, 255)
}

QPushButton::hover {
	background-color: rgb(62, 123, 255)
}</string>
         </property>
         <property name="text">
          <string>Create File</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="createdFile_Label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>630</y>
      <width>551</width>
      <height>0</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(0, 255, 0);
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="text">
     <string>Created file!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="allInputsMustBeFilled_Label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>630</y>
      <width>551</width>
      <height>0</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(197, 0, 0);
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="text">
     <string>Error: All inputs must be filled.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
